aten/src/ATen/cpu/FlushDenormal.cpp

```diff
@@ -0,0 +1,32 @@
+#include <ATen/cpu/FlushDenormal.h>
+
+#include <ATen/cpu/vec256/intrinsics.h>
+#include <cpuinfo.h>
+
+namespace at { namespace cpu {
+
+static constexpr unsigned int DENORMALS_ZERO = 0x0040;
+static constexpr unsigned int FLUSH_ZERO = 0x8000;
+
+bool set_flush_denormal(bool on) {
```

Comment:
nit: the API is a little unintuitive because both the return and parameter are bool, so I wasn't sure if the return is what the state is after vs if it succeeded.